src/modules/dashboard/components/ColonyFunding/ColonyFunding.css

```diff
@@ -49,3 +50,134 @@
   width: calc(100% - 60px);
   position: relative;
 }
+
+@media screen and query700 {
+  .main {
+    display: block;
+    margin-top: 0;
+  }
+
+  .content {
+    justify-content: flex-start;
+    height: auto;
+    gap: 20px;
+  }
+
+  .titleContainer {
+    align-items: center;
+    margin-bottom: 20px;
+    padding: 0 14px;
+    height: 50px;
+  }
+
+  .titleContainer h3 {
+    margin: 0;
+    width: auto;
+  }
+
+  .aside {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 0;
+  }
+
+  .banner {
+    margin: 0px;
+  }
```

Comment:
When there are multiple token entries on the page, making the page scroll, this element has too little bottom margin.

I think we should increase it.

![Screenshot from 2022-06-29 15-19-28](https://user-images.githubusercontent.com/1193222/176435100-f86f6267-6264-47e5-b619-212dba67c2c5.png)
![Screenshot from 2022-06-29 15-19-31](https://user-images.githubusercontent.com/1193222/176435107-6574bfa2-868a-4a40-aa5f-d0fbc3cf0cda.png)